app/popup.tsx

```diff
@@ -44,6 +44,15 @@ export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
     MessageService.getRequests(this.props.tabId, requests => {
       this.props.updateField("requests", requests);
     });
+
+    MessageService.getEnabledStatus(this.props.tabId, (enabledStatus:boolean) =>{
+      this.props.updateField("enabled", enabledStatus)
+    })
+
+    if (!(this.isUrlInValid(this.props.tabUrl))) {
+      this.props.updateField("errorMessage", "");
+      return;

```

Comment:
why this return?

Reply:
Suppose a user tries to Start Listening to requests on chrome:// pages, it gives us an error by checking whether `this.props.tabUrl` is a chrome URL. Now, since the redux state gets persisted across all tabs, this check is needed on `componentWillMount` for all tabs to remove the earlier error generated from this [line](https://github.com/code-mancers/interceptor/pull/47/files/48157dc30868f91f1b6f5bf2e64eb97c3fcc4f8c#diff-9744d706869f0ab0f5de1332df64040cR116)

Reply:
Yes, the `return` isn't required. Removed that 👍

Reply:
What ever the error is, we can clear the errorMessage when the user reopen the popup.
Its is not just meant for validURl. Hence we don't need a check there.